CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Changed
 
 - Moved the `solc` client to another crate at `era-solc`
+- Prohibited usage of the upstream `solc` compiler
```

Comment:
PR is huge, I'm not sure I see the error returned in case upstream `solc` is in use. May I ask you to point me to the place where it happens, please? Thanks!

Reply:
Ah, ok, I found it. GitHub collapsed them.